geoportailv3/static/js/maincontroller.js

```diff
@@ -61,7 +61,27 @@ app.MainController = function($scope, gettextCatalog, langUrlTemplate) {
   /**
    * @type {Boolean}
    */
-  this['infosOpen'] = false;
+  this['measureOpen'] = false;
+
+  /**
+   * @type {Boolean}
```

Comment:
The correct spelling is `boolean` rather than `Boolean` I think, but that may be convention: lowercase for primitive types (`number`, `string`, `boolean`, `function`, `null`, `undefined`), and uppercase for objects (`Object`, `Array`...).